internal/apigw/apigw_openapi_schema.go

```diff
@@ -72,6 +72,9 @@ func (sc *schemaContainer) Message(m pgs.Message, filter []string, nullable *boo
 		Properties: map[string]*dm_base.SchemaProxy{},
 		Nullable:   nullable,
 		Deprecated: deprecated,
+		Extensions: map[string]any{
+			"x-speakeasy-name-override": pgs.Name(*m.Descriptor().Name).UpperCamelCase().String(),
```

Comment:
hmm, not quite sure why you have to go to the descriptor to get this :)

Reply:
Changed to `m.Name()`